src/libomemo.c

```diff
@@ -1,3 +1,18 @@
+/**
+ * LIBOMEMO 0.5.0
```

Comment:
> LIBOMEMO 0.5.0

Is this string really required? You will need to update each file before each release...